### PR TITLE
WIP: Navigation

### PR DESCRIPTION
**Pull request by VilfredSikker**
Created Navigation from Login -> CreateSession & Add Item -> Start Lobby -> Start Session -> Estimate Item.

Note: Only navigation, no styling or functionality

### CODE REVIEW

File: PlanningPoker.App/Views/WelcomeScreen/Login.xaml.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public Login()
         private void HandleLoginClicked(object sender, EventArgs e)
         {
             this._loginViewModel.ExecuteLoginCommand();
+            this.Navigation.PushAsync(new CreateSession.CreateSession());
```
Comment (oliverjuhl) [inline PlanningPoker.App/Views/WelcomeScreen/Login.xaml.cs]: Ser lidt weird ud i forhold til de andre. Er det fordi du både laver et objekt samt kalder en metode på en linje eller hvad sker der her?

Comment (VilfredSikker) [reply PlanningPoker.App/Views/WelcomeScreen/Login.xaml.cs]: Det er måden den laver Reference på. Den skal ind i CreateSession(mappen) og ind i filen CreateSession

**Comment by VilfredSikker** [comment]
When we click Login at the moment, the navigation "ignores" wether or not it is allowed. So yes, the navigation should get a confirmation from the ViewModel. This was thought of as the "initial" navigation step. It is not optimized ;) @oliverjuhl

**Comment by oliverjuhl** [comment]
But 

> When we click Login at the moment, the navigation "ignores" wether or not it is allowed. So yes, the navigation should get a confirmation from the ViewModel. This was thought of as the "initial" navigation step. It is not optimized ;) @oliverjuhl

But why would we ignore this, when wer are ready for this feature?

**Comment by VilfredSikker** [comment]
> > When we click Login at the moment, the navigation "ignores" wether or not it is allowed. So yes, the navigation should get a confirmation from the ViewModel. This was thought of as the "initial" navigation step. It is not optimized ;) @oliverjuhl
> 
> But why would we ignore this, if we are ready for this feature?

Wasn't 100% sure we were. But I will implement it then.

File: PlanningPoker.App/Views/WelcomeScreen/Login.xaml.cs (hunk 1 above)
Comment (IMarnild) [reply PlanningPoker.App/Views/WelcomeScreen/Login.xaml.cs]: Måske det ikke er så smart at kalde mappen nøjagtigt det samme som klassen og vice versa.

**Pull request closed without merging**